src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
```

img tag - has display: inline; by default
So, height, width, and vertical margin don't work

```suggestion
    display: block;
    width: 160px;
    height: 134px;
    margin: 15px 5px 42px;
```